api/utils/index.js

```diff
@@ -5,6 +5,16 @@ const moment = require('moment');
 const config = require('../../config');
 const { logger } = require('../../winston');
 
+function generateS3ServiceName(owner, repository) {
```

Comment:
i realized that an owner could have a hyphen and therefore create a naming conflict. for example:

owner1: hello-awesome
repo1: people

vs.

owner2: hello
repo2: awesome-people

bothe create "hello-awesome-people"

Reply:
wordy but probably easier to work with ... especially when troubleshooting

Reply:
👍